code/game/objects/items/RCD.dm

```diff
@@ -276,6 +402,123 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	. = ..()
 	AddElement(/datum/element/openspace_item_click_handler)
 
+	//Create the radial menu as a static list only at the beginning of the map for all RCD instances to share
+	if(!mapload)
+		return
+
+	var/list/icon_list = list()
+	var/list/essentials = list(
+		'icons/hud/radial.dmi' = list("wallfloor", "windowsize", "windowtype", "grillewindow", "cnorth", "csouth", "ceast", "cwest", "chair", "stool", "windoor", "secure_windoor", "airlock", "airlocktype", "access", "computer_dir"),
+		'icons/obj/structures.dmi' = list("window0", "rwindow0", "table", "glass_table"),
+		'icons/obj/stock_parts.dmi' = list("box_1"),
+		'icons/obj/mining.dmi' = list("silo"),
+		'icons/obj/doors/airlocks/station/public.dmi' = list("closed")
+	)
+	var/icon/icon
+	for(var/icon_file as anything in essentials)
+		for(var/icon_state as anything in essentials[icon_file])
+			icon = icon(icon = icon_file, icon_state = icon_state)
+			if(icon_state == "closed")
+				icon.Blend(icon(icon = icon_file, icon_state = "fill_closed", dir = SOUTH), ICON_OVERLAY)
+				icon_list["airlock-solid"] = icon
+
+				icon = icon(icon = icon_file, icon_state = icon_state)
+				icon_list["airlock-glass"] = icon
+			else
+				icon = icon(icon = icon_file, icon_state = icon_state)
+				if(icon_state == "window0" || icon_state == "rwindow0")
```

Comment:
All of this is so hackyyyy, document why you're doing any of this

Reply:
This is "window0"
![Screenshot (57)](https://user-images.githubusercontent.com/110812394/205889462-3992f363-ac55-4a17-88df-48b595794b98.png)

This is grill
![Screenshot (58)](https://user-images.githubusercontent.com/110812394/205889513-4324b34b-9f87-4bdf-80b5-30455c300998.png)

When i blend them together i get this[scaled as i took screen shot from ingame]
![Screenshot (59)](https://user-images.githubusercontent.com/110812394/205890131-202ab188-434f-4fa4-b8e3-17028ea04a7e.png)

Which is the icon i use for Fulltile window

Similarly, "rwindow0" is for Refinforced Fulltile window

I'll put a comment saying that

Also this code is part of the radial menu. Still waiting on your word to scrap this